tests/foreman/ui/test_host.py

```diff
@@ -536,7 +545,284 @@ def test_positive_update_with_user(self):
+            associate org.location, and if its not there then creates
+            new.
+        10. Search 'Kickstart default' partition table and RH Atomic OS along
+            with PXE templates.
+        11. Associates org, location and OS with provisioning and PXE templates
+        12. Search for x86_64 architecture
+        13. Associate arch, partition table, provisioning/PXE templates with OS
+        14. Search for existing Atomic media or create new otherwise and
+            associate org/location
+        15. Create new host group with all required entities
+        """
+        super(AtomicHostTestCase, cls).setUpClass()
+        # Create a new Organization and Location
+        cls.org = entities.Organization(name=gen_string('alpha')).create()
+        cls.org_name = cls.org.name
+        cls.loc = entities.Location(
+            name=gen_string('alpha'),
+            organization=[cls.org],
+        ).create()
+        cls.loc_name = cls.loc.name
+        # Create a new Life-Cycle environment
+        cls.lc_env = entities.LifecycleEnvironment(
+            name=gen_string('alpha'),
+            organization=cls.org,
+        ).create()
+        cls.rh_ah_repo = {
+            'name': REPOS['rhaht']['name'],
+            'product': PRDS['rhah'],
+            'reposet': REPOSET['rhaht'],
+            'basearch': None,
+            'releasever': None,
+        }
+        with manifests.clone() as manifest:
+            upload_manifest(cls.org.id, manifest.content)
+        # Enables the RedHat repo and fetches it's Id.
+        cls.repo_id = enable_rhrepo_and_fetchid(
+            basearch=cls.rh_ah_repo['basearch'],
+            # OrgId is passed as data in API hence str
+            org_id=str(cls.org.id),
+            product=cls.rh_ah_repo['product'],
+            repo=cls.rh_ah_repo['name'],
+            reposet=cls.rh_ah_repo['reposet'],
+            releasever=cls.rh_ah_repo['releasever'],
+        )
+        # Increased timeout value for repo sync
+        cls.old_task_timeout = entity_mixins.TASK_TIMEOUT
+        entity_mixins.TASK_TIMEOUT = 600
+        # Sync repository
+        entities.Repository(id=cls.repo_id).sync()
+        cls.cv = entities.ContentView(organization=cls.org).create()
+        cls.cv.repository = [entities.Repository(id=cls.repo_id)]
+        cls.cv = cls.cv.update(['repository'])
+        cls.cv.publish()
+        cls.cv = cls.cv.read()
+        promote(cls.cv.version[0], cls.lc_env.id)
+        # Search for SmartProxy, and associate location
+        cls.proxy = entities.SmartProxy().search(
+            query={
+                u'search': u'name={0}'.format(
+                    settings.server.hostname
+                )
+            }
+        )[0]
+        cls.proxy.location = [cls.loc]
+        cls.proxy = cls.proxy.update(['location'])
+        cls.proxy.organization = [cls.org]
+        cls.proxy = cls.proxy.update(['organization'])
```

Comment:
`cls.proxy = cls.proxy.update(['organization', 'location'])`
and remove `cls.proxy = cls.proxy.update(['location'])` should work